components/OverviewCard/index.js

```diff
@@ -1,17 +1,28 @@
 import styled from "styled-components";
 import Link from "next/link";
 import { formatToEUR } from "@/helper/formatToEUR";
+import "chart.js/auto";
+import { Pie } from "react-chartjs-2";
+import { chartOptions } from "@/helper/chartOptions";
+import { createChartData } from "@/helper/createChartData";
+import useMedia from "@/hooks/useMedia";
+import { sumTotalPrice } from "@/helper/sumTotalPrice";
 
 export default function OverviewCard({ project }) {
-  const sumUpArray = (accumulator, currentValue) => accumulator + currentValue;
-  const totalPrice = project.items
-    .map((item) => Number(item.price))
-    .reduce(sumUpArray, 0);
-
+  const labelColor = useMedia(["(prefers-color-scheme: dark)"], [true], false)
```

Comment:
I don't understand how this hook works, but I guess it works as it's supposed to.

Reply:
u r right the useMedia hook is overkill, i will change it to isDarkMode because i don't plan user interaction with it. thx!